inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/InlongGroupContext.java

```diff
@@ -75,6 +83,22 @@ public InlongGroupContext(InnerGroupContext groupContext) {
         }
     }
 
+    private List<StreamSource> getGroupSources() {
+        List<StreamSource> groupSources = Lists.newArrayList();
+        this.inlongStreamMap.values().forEach(inlongStream -> {
+            Map<String, StreamSource> sources = inlongStream.getSources();
+            if (MapUtils.isNotEmpty(sources)) {
+                for (Map.Entry<String, StreamSource> entry : sources.entrySet()) {
+                    StreamSource source = entry.getValue();
+                    if (source != null) {
+                        sourcesInGroup.add(source);
```

Comment:
Where is the "sourcesInGroup" comes from?

Reply:
Out of no where.